src/pages/@owner/estimate/@section/basic-info-section/Price.tsx

```
@@ -0,0 +1,23 @@
+import FormLayout from '@components/layout/form-layout/FormLayout';
+import Input from '@ui/input/Input';
+import ErrorText from '@form/error-text/ErrorText';
+import { formatPrice } from '@utils/price-formatter';
+
+interface PriceProps {
+  price: number;
+  updatePrice: (price: string) => void;
```

ESLint

[error] 'price' is defined but never used. Allowed unused args must match /^_/u.

(no-unused-vars)